contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -4,10 +4,8 @@ import {Registry} from "../../common/Registry.sol";
 
 import {IValidatorShare} from "./IValidatorShare.sol";
 
-// TODO: refactor each function to reusable smaller internal functions
+
 contract ValidatorShare is IValidatorShare {
-    // TODO: totalStake and active ammount issue
```

Comment:
Not sure whether we should be resolving these todos or removing them? 3 others in this file.

Reply:
this one is resolved, `totalStake` is not used for any real accounting for rewards etc.used to keep track of total stake locked in the contract for external reads